dbtvault-dev/macros/materialisations/vault_insert_by_rank_materialization.sql

```diff
@@ -126,7 +126,7 @@
         {% endif %}
 
         {% call noop_statement('main', "BASE LOAD {}".format(rows_inserted)) -%}
-            {{ build_sql }}
```

Comment:
These will also need to be restored

Reply:
Why?

Reply:
It forces dbt to output the generated SQL in the compiled and run folders, which is good for debugging and general auditability etc.